tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Support/ScenarioContextKeys.cs

```diff
@@ -1,9 +1,10 @@
-﻿namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Steps.Support
-{
-    internal static class ScenarioContextKeys
-    {
-        internal const string AccessToken = nameof(AccessToken);
-        internal const string AddressMapDictionary = nameof(AddressMapDictionary);
-        internal const string ContactMapDictionary = nameof(ContactMapDictionary);
-    }
-}
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Steps.Support
+{
+    internal static class ScenarioContextKeys
+    {
+        internal const string AccessToken = nameof(AccessToken);
+        internal const string AddressMapDictionary = nameof(AddressMapDictionary);
+        internal const string ContactMapDictionary = nameof(ContactMapDictionary);
+        internal const string PayloadDictionary = nameof(PayloadDictionary);

```

Comment:
The name of the variable is a little generic. Could this be something more specific? e.g. OrganisationPayloadDictionary / OrderingPartyPayloadDictionary. This would provide additional context when reading the code & hopefully flag up any incorrect use within an other steps class in the future.